### ISSUE TITLE
update 'quotes' rule for 3.0

### ISSUE THREAD

**Issue by MelSumner** [ember-template-lint/ember-template-lint#1712]
For 'next' branch: 

Now that `.html` files are included, the `quotes` rule (you must use double quotes in templates) is throwing a false positive over acceptable lines. 

Example: 

```hbs
{{content-for 'head'}}
{{content-for 'head-footer'}}
```

These should be allowed.

**Comment by rwjblue** [comment]
Hmm, I don't think we should allow those arbitrarily. Applications are manually opting in to "stylistic" configuration, which requires double quotes (Note: it is not in default recommended set). 

I also think it would be perfectly fine to update the ember-cli blueprint to use double quotes in the blueprint (to reduce the manual changes needed for new projects that want to use stylistic config).

**Comment by MelSumner** [comment]
ok, I'll close this as not a bug then. Thank you for the feedback!
